src/main/java/com/spring/app/controllers/InitDBController.java

```diff
@@ -0,0 +1,120 @@
+package com.spring.app.controllers;
+
+import com.spring.app.CountryLevel;
+import com.spring.app.domain.Company;
+import com.spring.app.domain.Country;
+import com.spring.app.domain.Employee;
+import com.spring.app.domain.Vacancy;
+import com.spring.app.repos.CompanyRepository;
+import com.spring.app.repos.CountryRepository;
+import com.spring.app.repos.EmployeeRepository;
+import com.spring.app.repos.VacancyRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+
+import java.util.Date;
+import java.util.Map;
+
+/**
+ * InitDBController
+ *
+ * @author lyubov
+ */
+public class InitDBController {
+
+    @Autowired
+    private VacancyRepository vacancyRepository;
+    @Autowired
+    private CompanyRepository companyRepository;
+    @Autowired
+    private CountryRepository countryRepository;
+    @Autowired
+    private EmployeeRepository employeeRepository;
+
+    @PostMapping("newDBinit")
+    public ResponseEntity filter(Map<String, Object> model){
```

Comment:
use @PostConstruct for data init with a ddl-auto: create

Reply:
Can you explain me this moment later? I want to discuss it.

Reply:
No!